package.json

```diff
@@ -13,6 +13,7 @@
     "url": "https://github.com/Salesforce/refocus.git"
   },
   "scripts": {
+    "cleanup": "npm run dropdb && npm run initdb && npm run checkdb",
```

Comment:
thx for adding this--i got rid of stuff a little too aggressively a couple of weeks ago ;)